resources/css/_props.css

```diff
@@ -33,16 +33,16 @@
 
     /* Link */
     --link-color: var(--theme-link-color, var(--color-blue-6));
-    --link-color-hover: var(--theme-link-color-hover, var(--color-blue-7));
-    --link-color-visited: var(--theme-link-color-visited, var(--color-blue-8));
+    --link-color-hover: var(--theme-link-color-hover, var(--color-blue-8));
+    --link-color-visited: var(--theme-link-color-visited, var(--color-blue-7));
     --link-outline-focus: var(--theme-link-outline-focus, var(--color-blue-6));
     --link-color-active: var(--theme-link-color-active, var(--color-blue-7));
     --link-background-active: var(--theme-link-background-active, var(--color-grey-2));
 
     /* Link on dark backgrounds */
     --link-dark-color: var(--theme-link-dark-color, var(--color-white));
-    --link-dark-color-hover: var(--theme-link-dark-color-hover, var(--color-turquoise-1));
-    --link-dark-color-visited: var(--theme-link-dark-color-visited, var(--color-turquoise-1));
+    --link-dark-color-hover: var(--theme-link-dark-color-hover, var(--color-grey-1));
+    --link-dark-color-visited: var(--theme-link-dark-color-visited, var(--color-turquoise-2));
```

Comment:
I don't believe these styles are for dark mode, rather they are for dark backgrounds such as the darker areas on the welcome page. These changes actually should be applied to [_dark.css](https://github.com/accessibility-exchange/platform/blob/e2afafbaf8d2169263b7e1913cd2b11ac839c656/resources/css/themes/_dark.css#L34-L35)